packages/layout/styles/main.css

```
@@ -7,11 +14,113 @@
     width: 100%;
   }
 
-  footer {
-    margin-top: 100px;
+  /* Container Query Contexts */
+  #header,
+  #footer,
+  .content-area {
+    container-type: inline-size;
+  }
+
+  #header {
+    container-name: header;
+  }
+
+  #footer {
+    container-name: footer;
   }
 
-  .breadcrumbs {
-    padding-bottom: 1em;
+  .content-area {
+    container-name: content;
+  }
+
+  /* Blocks & Inner containers */
+  #header,
+  #footer {
+    [class*='section'] {
+      & .section-inner-container {
+        display: grid;
+        width: 100%;
+        max-width: var(--block-width, 100%);
+        align-items: var(--align-items, center);
+        margin: 0 auto;
+        grid-auto-flow: var(--grid-auto-flow, row);
+        justify-items: var(--justify-items, start);
+      }
+
+      &.layout {
+        --block-width: var(--layout-container-width);
+      }
+
+      &.default {
+        --block-width: var(--default-container-width);
+      }
+
+      &.narrow {
+        --block-width: var(--narrow-container-width);
+      }
+    }
+  }
+
+  .content-area {
+    [class*='block'] {
```

```suggestion
    .block {
```

I would just go with this